src/DataViews/FunctionsDataViewTest.cpp

```diff
@@ -0,0 +1,655 @@
+
+  // Context menus when multiple entries are selected: A particular action will be offered when any
+  // selected entry can execute that action.
+  is_function_selected = {false, false, false};
+  is_frame_track_enabled = {false, false, false};
+  EXPECT_THAT(view_.GetContextMenu(0, {0, 1, 2}), can_hook());
+  EXPECT_THAT(view_.GetContextMenu(0, {0, 1, 2}), can_enable_frame_tracks());
+
+  is_function_selected = {false, true, false};
+  is_frame_track_enabled = {false, false, false};
+  EXPECT_THAT(view_.GetContextMenu(0, {0, 1, 2}), can_hook_and_unhook());
+  EXPECT_THAT(view_.GetContextMenu(0, {0, 1, 2}), can_enable_frame_tracks());
+
+  is_function_selected = {false, true, false};
+  is_frame_track_enabled = {false, true, false};
+  EXPECT_THAT(view_.GetContextMenu(0, {0, 1, 2}), can_hook_and_unhook());
+  EXPECT_THAT(view_.GetContextMenu(0, {0, 1, 2}), can_enable_and_disable_frame_tracks());
+
+  is_function_selected = {false, true, true};
+  is_frame_track_enabled = {false, true, false};
+  EXPECT_THAT(view_.GetContextMenu(0, {0, 1, 2}), can_hook_and_unhook());
+  EXPECT_THAT(view_.GetContextMenu(0, {0, 1, 2}), can_enable_and_disable_frame_tracks());
+}
+
+TEST_F(FunctionsDataViewTest, GenericDataExportFunctionShowCorrectData) {
+  // This functionality is not tested in this test case.
+  EXPECT_CALL(app_, IsFunctionSelected)
+      .Times(testing::AnyNumber())
+      .WillRepeatedly(testing::Return(false));
+
+  // This functionality is not tested in this test case.
+  EXPECT_CALL(app_, IsFrameTrackEnabled)
+      .Times(testing::AnyNumber())
+      .WillRepeatedly(testing::Return(false));
+
+  // This functionality is not tested in this test case.
+  EXPECT_CALL(app_, HasCaptureData)
+      .Times(testing::AnyNumber())
+      .WillRepeatedly(testing::Return(false));
+
+  view_.AddFunctions({&functions_[0]});
+
+  std::vector<std::string> context_menu = view_.GetContextMenu(0, {0});
+
+  // Copy Selection
+  {
+    const auto copy_selection_idx =
+        std::find(context_menu.begin(), context_menu.end(), "Copy Selection") -
+        context_menu.begin();
+    ASSERT_LT(copy_selection_idx, context_menu.size());
+
+    std::string clipboard;
+    EXPECT_CALL(app_, SetClipboard).Times(1).WillOnce(testing::SaveArg<0>(&clipboard));
+    view_.OnContextMenu("Copy Selection", static_cast<int>(copy_selection_idx), {0});
+    EXPECT_EQ(clipboard, absl::StrFormat(
+                             "Hooked, Function, Size, Module, Address in module\n"
+                             ", %s, %d, %s, %#x\n",
+                             functions_[0].pretty_name(), functions_[0].size(),
+                             std::filesystem::path{functions_[0].module_path()}.filename().string(),
+                             functions_[0].address()));
+  }
+
+  // Export to CSV
+  {
+    const auto export_to_csv_idx =
+        std::find(context_menu.begin(), context_menu.end(), "Copy Selection") -
```

Comment:
I think this was wrongly copied from the test before. The string should be "Export to CSV". 

Researching why this test succeeds, even with this wrong string, led me to the following: The parameter `menu_index` in `DataView::OnContextMenu()` seems to be not used at all in the code base. Maybe consider removing it.

Reply:
Yeah, I won't change any code from the data views here. Otherwise I won't be finished for Christmas. :innocent: But I will fix the literal. Thanks for spotting this.

Reply:
Done.